dts/arm/yaml/st,stm32-ccm.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Would require a rebase as dts/arm/yaml no more exists.
Should be moved under dts/bindings/ccm (/tcm?)

Reply:
I overlooked that the yaml file was now a completely new file, I will look into it tonight after work.

Any comment on the general setup ?  Is it ok like this by doing it with dts and zephyr  choose ccm0 =  &ccm ?